FAQUse: what is a coinjoin strategy (#1301)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

Depending on many factors, like the `Anonymity score target`, the `coinjoin strategy`, and the amount of bitcoin, it can take from a couple hours to multiple days.
:::

::::details
### What is a coinjoin strategy?

A coinjoin strategy has specific pre-configured coinjoin settings.

There are multiple coinjoin settings which the user is able to configure according to their own prefence.
Each coinjoin strategy has different specific settings for optimal performance.

> After a coinjoin strategy is selected, the according settings are applied.

Reviewer: ```suggestion
After a coinjoin strategy is selected, its settings are applied.
```
